lib/components/popover.vue

```diff
@@ -123,7 +123,7 @@
                 const cfg = assign({}, this.baseConfig);
                 if (this.$refs.title.innerHTML.trim()) {
                     // We pass the DOM element to preserve components
-                    cfg.title = this.$refs.title;
+                    cfg.title = this.$refs.title.innerHTML;
```

Comment:
If we do this, we can't pass components to the title (i.e. say someone wants top place a close button in the title)

Reply:
okay. agree